Firebase.podspec

```diff
@@ -37,7 +37,7 @@ Simplify your app development, grow your user base, and monetize more effectivel
     ss.ios.dependency 'FirebaseAnalytics', '~> 8.12.0'
```

Comment:
This file should be deleted along with `Firebase.h`.

Reply:
Fixed